packages/core/package.json

```
@@ -43,7 +43,7 @@
     "@blitzjs/config": "0.15.3",
     "@blitzjs/display": "0.15.3",
     "pretty-ms": "6.0.1",
-    "react-query": "1.5.8",
+    "react-query": "^2.4.4",
```

This needs pinned to an exact version :)
```suggestion
    "react-query": "2.4.4",
```

---

Fixed